libcontainer/init_linux.go

```diff
@@ -71,6 +73,108 @@ type initConfig struct {
 	Cgroup2Path      string                `json:"cgroup2_path,omitempty"`
 }
 
+// StartInitialization loads a container by opening the pipe fd from the parent
+// to read the configuration and state. This is a low level implementation
+// detail of the reexec and should not be consumed externally.
+func StartInitialization() (err error) {
```

Comment:
not new, but can we rename the named error return? It's hard to see if it's potentially shadowed in various locations; (`newVar, err := something()`)

Or we should rewrite some code to explicitly declare variables to be a bit more defensive

Reply:
Done, thanks for the suggestion -- I think this made the code a tad cleaner.